core/state_processor_test.go

```diff
@@ -17,8 +17,11 @@
 package core
 
 import (
+	//"bytes"
```

Comment:
TODO: remove the block extraction code, maybe extract and move this to an external utility